core/src/main/web/app/mainapp/components/notebook/codecell.jst.html

```diff
@@ -36,6 +36,7 @@
         <div class="cell-power-button" title="initialization cell is on" data-ng-class="{'visible': isInitializationCell()}"><i class="fa fa-power-off"></i></div>
       </div>
     </div>
+    <div data-ng-if="!displayOutput || !hasOutput()" data-ng-style="{'min-height': (cellmodel.output && cellmodel.output.height) + 'px'}"></div>
```

Comment:
Good thinking about data- prefixing but we don't support any old browsers so let's just remove the prefixes so they don't cause confusion.